notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>For readability, I think you can remove the <code>IS_TESTING</code> check, but it is completely reasonable to have it here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Since the testing environment doesn't use a cluster and switching a kernel, I had to added them.</p>